src/fu-util-common.c

```diff
@@ -2164,6 +2164,33 @@ fu_util_remote_to_string(FwupdRemote *remote, guint idt)
 	return g_string_free(g_steal_pointer(&str), FALSE);
 }
 
+const gchar *
+fu_util_request_get_message(FwupdRequest *req)
```

Comment:
as a thought, I wonder if these should be emitted from libfwupd?  Then it's easier on GUI clients?

Reply:
I'd really like frontends to use the gnome/kde specific themed icons where applicable.

Reply:
But for the message string, wouldn't a helper be good for them to pick up from?

Reply:
OK